auth/src/main/res/values/styles.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
 
-    <style name="FirebaseUI" parent="Theme.AppCompat.Light.DarkActionBar">
+    <style name="FirebaseUI" parent="Theme.AppCompat.Light.NoActionBar">
```

Comment:
Parent should have DarkActionBar. No actionbar should be only in the first activity.